### ISSUE TITLE
Configuration and Input documentation

### ISSUE THREAD

**Issue by xenocons** [fsprojects/FSharp.Data.SqlClient#236]
Under 'SqlProgrammabilityProvider' on http://fsprojects.github.io/FSharp.Data.SqlClient/configuration%20and%20input.html it lists ResolutionFolder as type parameter, however based upon https://github.com/fsprojects/FSharp.Data.SqlClient/blob/master/src/SqlClient/SqlClientProvider.fs#L42 this does not seem to be a parameter, is there something I am missing?

**Comment by dsevastianov** [comment]
Good point, it is sourced from TypeProviderConfig now, we have to update documentation.
